### PR TITLE
Release kit software v2022.1.0

### PR DESCRIPTION
**Pull request by trickeydan**
~**Please do not merge. Approve if happy and I will merge once the image is uploaded**~

Files are all uploaded. Merging this PR will release to competitors.

### CODE REVIEW

File: _data/kit_versions.yml [hunk 1]
```diff
@@ -4,3 +4,5 @@
   released: 2021-11-20
 - version: 2022.0.0
   released: 2021-11-13
+- version: 2022.1.0
```
Comment (RealOrangeOne) [inline _data/kit_versions.yml:7]: New versions should be added to the top of the file, not the bottom (perhaps we should change that)

Comment (trickeydan) [reply _data/kit_versions.yml]: Fixed

File: programming/python/libraries.md [hunk 2]
```diff
@@ -23,22 +23,23 @@ The following python libraries are installed and available for use in your robot
...
 * [kiwisolver 1.3.2](https://pypi.org/project/kiwisolver)
 * [matplotlib 3.4.1](https://pypi.org/project/matplotlib)
 * [numpy 1.21.2](https://pypi.org/project/numpy)
 * opencv 4.5.2
 * [pandas 1.3.3](https://pypi.org/project/pandas)
+* [pillow 8.3.2](https://pypi.org/project/Pillow/)
 * pkg-resources 57.4.0
 * [pydantic 1.8.2](https://pypi.org/project/pydantic)
-* [pygobject 3.40.1](https://pypi.org/project/PyGObject/)
+* [dateutil 2.8.2](https://pypi.org/project/python-dateutil/)
```
Comment (sedders123) [inline programming/python/libraries.md:35]: Looks like dateutil is listed here three times, is that intentional?

Comment (sedders123) [reply programming/python/libraries.md]: Have `pygobject` and `pyyaml` been removed from the image (these are what git says used to be on these lines)? If so all looks good to me!

**Review by raccube — APPROVED** [review]

**Pull request merged**